### PR TITLE
PDA: The Spammening

### PR DESCRIPTION
**Pull request by Cerebulon**
Greatly expands upon and tweaks PDA Spam messages for increased lore friendliness and shits n giggles.

Basically just a bunch more randomized elements based on IC media and products and so on and so forth, while removing real-world jokes like WoW Gold and Nigerian Princes. They're cats now. Meow. Maxman is also gender neutral in the stupidest autogenerated spam way possible.

### CODE REVIEW

File: code/modules/gamemaster/event2/events/everyone/pda_spam.dm [hunk 1]
```diff
@@ -78,55 +78,67 @@
...
 			"We have been filed with a complaint from one of your customers in respect of their business relations with you.",\
 			"We kindly ask you to open the COMPLAINT REPORT (attached) to reply on this complaint..")
 		if(4)
 			sender = pick("Buy Dr. Maxman","Having dysfuctional troubles?")
 			message = pick("DR MAXMAN: REAL Doctors, REAL Science, REAL Results!",\
-			"Dr. Maxman was created by George Acuilar, M.D, a [using_map.boss_short] Certified Urologist who has treated over 70,000 patients sector wide with 'male problems'.",\
-			"After seven years of research, Dr Acuilar and his team came up with this simple breakthrough male enhancement formula.",\
+			"Dr. Maxman was created by George Acuilar, M.D, a [using_map.boss_short] Certified Urologist who has treated over 70,000 patients region wide with '[pick("male","female", "other")] problems'.",\
+			"After seven years of research, Dr Acuilar and his team came up with this simple breakthrough [pick("male","female", "other")] enhancement formula.",\
 			"Men of all species report AMAZING increases in length, width and stamina.")
```
Comment (elgeonmb) [inline code/modules/gamemaster/event2/events/everyone/pda_spam.dm:111]: true gender equality when,,

Comment (Cerebulon) [reply code/modules/gamemaster/event2/events/everyone/pda_spam.dm:111]: true...

File: code/modules/gamemaster/event2/events/everyone/pda_spam.dm [hunk 2]
```diff
@@ -78,55 +78,69 @@
...
 			"Once you are a player with MaxBet, you will also receive lucrative weekly and monthly promotions.",\
+			"Life-changing winnings* await when you register with MaxBet today.",\
 			"You will be able to enjoy over 450 top-flight casino games at MaxBet.")
 		if(2)
-			sender = pick(300;"QuickDatingSystem",200;"Find your russian bride",50;"Tajaran beauties are waiting",50;"Find your secret skrell crush",50;"Beautiful unathi brides")
+			sender = pick(300;"QuickDatingSystem",200;"Find your almachi bride",50;"Tajaran beauties are waiting",50;"Find your secret skrell crush",50;"Beautiful unathi brides")
 			message = pick("Your profile caught my attention and I wanted to write and say hello (QuickDating).",\
-			"If you will write to me on my email [pick(first_names_female)]@[pick(last_names)].[pick("ru","ck","tj","ur","nt")] I shall necessarily send you a photo (QuickDating).",\
+			"If you will write to me on my email [pick(pick(first_names_female),(pick(first_names_male)))]@[pick(last_names)].[pick(".xo.vr","ck","tj","ur","gov","nt","xo.sh")] I shall necessarily send you a photo (QuickDating).",\
```
Comment (elgeonmb) [inline code/modules/gamemaster/event2/events/everyone/pda_spam.dm:92]: will result in ``..xo.vr`

**Review by Sypsoti — APPROVED** [review]
Finally, spam worth reading.

**Pull request merged**